example.config.yml

```diff
@@ -2,6 +2,13 @@
 # Location where configuration files will be stored.
 config_dir: '~'
 
+# Domain names configuration (related services need to be enabled).
+domain_name_enable: false
+domain_name: 'home.local'
+pihole_domain_name: 'pihole'          # to access pihole via: http://pihole.home.local
```

Comment:
I'd rather see these changed to:

```yaml
domain_pihole: 'pihole'
domain_grafana: 'grafana'
(etc)
```

So that all the variables are easier to lump together.

Reply:
I addressed your comments and pushed the changes. Please check again. Thanks